src/libraries/Microsoft.PowerFx.Core/Public/IPowerFxScope.cs

```diff
@@ -21,5 +21,12 @@ public interface IPowerFxScope
         /// Provide intellisense for expression
         /// </summary>
         IIntellisenseResult Suggest(string expression, int cursorPosition);
+
+        /// <summary>
+        /// Provider quick fix suggesions.
+        /// </summary>
+        /// <param name="expression">The formula expression.</param>
+        /// <returns>Collection of quick fixes.</returns>
+        QuickFixResult[] QuickFix(string expression);
```

Comment:
>expression [](http://example.com/codeflow?start=41&length=10)

does this need a cursorPosition too? (like intellisense?)

Reply:
PROSE repair engine doesn't need cursor position as it needs whole expression for repair suggestion and going to return entire formula to replace with. As per my understanding might not be useful at this moment.